GuiltMotivator/app/src/main/java/nielsen/guiltmotivator/MainActivity.java

```diff
@@ -132,25 +136,25 @@ public void onRadioButtonClicked(View v) {
         // all the radio buttons
         switch(v.getId()) {
             // if tone, make a snarky comment/toast and put in sharedPrefs
-            case R.id.polite:
-                if (v.getId() == R.id.polite) {
+            case R.id.politeButtonContact:
+                if (v.getId() == R.id.politeButtonContact) {
                     Toast.makeText(this, "Jolly good", Toast.LENGTH_SHORT).show();
-                    editor.putString(MainActivity.SAVED_TONE, "polite");
-                    editor.apply();
+//                    editor.putString(MainActivity.SAVED_TONE, "polite");
+//                    editor.apply();
```

Comment:
why are these commented out lines still here? just so we remember how to put things in sharedPrefs?